library/cohesity_job.py

```diff
@@ -479,7 +480,7 @@ def register_job(module, self):
                 vms = module.params.get('exclude')
                 payload['excludeSourceIds'] = get_vmware_ids(module, parent_source_id, self, vms)
         data = json.dumps(payload)
-        # module.exit_json(output=data)
+        # module.exit_json(output=payload)
```

Comment:
shouldn't this be json output instead of the object ?

Reply:
@pyashish these are comments for debugging, payload contains payload for API request in json format.